about us/about_us.css

```diff
@@ -6,3 +6,144 @@
   box-sizing: border-box;
   font-family: "Roboto", sans-serif;
 }
+
+body {
+  margin: 30px;
+  line-height: 25px;
+  text-indent: 15px;
+}
+
+.hide {
+  display: none;
+}
+
+h2 {
+  text-indent: 30px;
+  margin-top: 25px;
+  margin-bottom: 15px;
+}
+
+input[type="checkbox"] {
+  filter: grayscale(1);
+  margin-top: 6px;
+  margin-left: 6px;
+}
+
+.candidate-trait {
+  display: flex;
+}
+
+.candidate-traits {
+  margin-bottom: 15px;
+}
+
+.p-underline {
+  text-decoration: underline;
+  font-size: 20px;
+}
+
+.program-stages {
+  display: flex;
+  margin-top: 20px;
+}
+
+.programStageBox {
```

Comment:
you are using different naming conventions kabob-case and camelCase, you should pick one and stay consistent. I personally prefer camelCase, like programStageBox and not candidate-traits

Reply:
I decided to use kabob-case only for classes and ids